src/index.tsx

```diff
@@ -356,6 +356,28 @@ type AppState = {
   exportPadding: number;
 };
 
+const KEYS = {
+  ARROW_LEFT: "ArrowLeft",
+  ARROW_RIGHT: "ArrowRight",
+  ARROW_DOWN: "ArrowDown",
+  ARROW_UP: "ArrowUp",
+  ESCAPE: "Escape",
+  DELETE: "Delete",
+  BACKSPACE: "Backspace"
+};
```

Comment:
What's the rationale for this?

Reply:
The general premise is partially the same reason that best practice is to avoid magic numbers.

By assigning these values to a object and referencing them in this way it reduces the likeliness for a bug to occur when you reference the same value in multiple places. For instance, it would be easy to introduce a bug by accidentally having a typo where in one play you reference `ArrowLeft`, and in another place reference `ArrwoLeft`. Chances are if this type of thing did happen, it wouldn't be easy to spot. This also follows the DRY principal of "don;t repeat yourself".

Reply:
I agree with you. For me it's ok to merge.

Reply:
I also agree with his reason. I would also like to move the elementTypes to a enumerate. That way it's easier to know what kind of elementTypes we currently have, and easier to add/change any of them if necessary.